ZypeAppleTVBase/Utilities/SSUtils.swift

```diff
@@ -40,6 +40,24 @@ open class SSUtils {
         return dateFormatter.date(from: string)
     }
     
+    open static func arrayFromDictionary(_ dic: Dictionary<String, AnyObject>?, key: String) throws -> Array<AnyObject>
```

Comment:
Couple of minor review comments for betterment of Zype codebase - 

1. We must not use forced unwrap (!) in production codebase. This is not recommended and I have already created a task to remove forced unwrap completely from Zype codebase.
2. This 17 lines functions can be written only in 7 lines as shown below, we should prefer latest swift programming concepts to write optimised and crisp code if possible. 
3. I know existing code is using forced unwrap and not optimised but i always prefer to write new swift code considering latest programming tips and optimised.

open static func arrayFromDictionary(_ dic: Dictionary<String, AnyObject>?, key: String) throws -> Array<AnyObject>
    {
        guard let value = dic?[key] as? Array<AnyObject> else {
            throw UtilError.invalidArgument
        }

        return value
    }